data-prepper-plugins/http-common/src/main/java/org/opensearch/dataprepper/plugins/server/CreateServer.java

```diff
@@ -163,17 +243,17 @@ public Server createHTTPServer(final Buffer<Record<Log>> buffer, final Certifica
             // TODO: enable encrypted key with password
             sb.https(serverConfiguration.getPort()).tls(
                     new ByteArrayInputStream(certificate.getCertificate().getBytes(StandardCharsets.UTF_8)),
-                    new ByteArrayInputStream(certificate.getPrivateKey().getBytes(StandardCharsets.UTF_8)
-                    )
-            );
+                    new ByteArrayInputStream(certificate.getPrivateKey().getBytes(StandardCharsets.UTF_8)));
         } else {
             LOG.warn("Creating http source without SSL/TLS. This is not secure.");
-            LOG.warn("In order to set up TLS for the http source, go here: https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/http-source#ssl");
+            LOG.warn(
+                    "In order to set up TLS for the http source, go here: https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/http-source#ssl");
```

Comment:
This error message can be misleading. We are in the otel-source not the http source, right?

Reply:
Please, @Davidding4718 @dlvenable  Let me know your thoughts on this as the original authors. I can do a minor PR for this.

Reply:
Created the PR here: https://github.com/opensearch-project/data-prepper/pull/5689

Reply:
Approved. Might cause merge conflicts though. Be careful, not to take it out again.